worlds/sa2b/CHANGELOG.md

```diff
@@ -59,6 +59,8 @@
 - Going into submenus from the pause menu should no longer reset traps
 - `Sonic - Magic Gloves` are now plural
 - Junk items will no longer cause a crash when in a falling state
+- Saves should no longer incorrectly be marked as not matching the connected server.
```

Comment:
```suggestion
- Saves should no longer incorrectly be marked as not matching the connected server
```
Most of the other lines don't end with a ., so the . should be removed for consistency, or added to all the others.